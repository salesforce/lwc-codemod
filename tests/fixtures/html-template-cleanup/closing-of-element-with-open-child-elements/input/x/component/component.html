<template>
    <section>
        <span>hello</span>
        <span>mr. anderson</span>
</template>
