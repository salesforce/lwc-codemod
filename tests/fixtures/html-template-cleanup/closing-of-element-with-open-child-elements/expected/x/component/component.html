<template>
    <section>
        <span>hello</span>
        <span>mr. anderson</span>
</section></template>
